features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1518,6 +1518,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
     const updateConfigurationForSPA = (values: any, url?: string, origin?: string): any => {
```

Comment:
Don't we need to update the functions for other application types too?

Reply:
This issue is specific to SPAs. Hence we are only fixing for the SPAs.